### ISSUE TITLE
iOS top safe area

### ISSUE THREAD

**Issue by ferrannp** [okwasniewski/react-native-bottom-tabs#26]
<img width="348" alt="image" src="https://github.com/user-attachments/assets/9223d28c-7a0c-4e24-85de-49eeab70a1e4">

Adding tabs changes the background of the top safe area (in this screenshots should be grey, not white).

**Comment by okwasniewski** [comment]
Hey, version 0.0.9 introduces `ignoresTopSafeArea` prop for the navigator this should extend the view by ignoring safe area

**Comment by EvanBacon** [comment]
Screen props aren't passed in the standard React Navigation way, if you're using Expo Router then pass the option to the top-level navigator like so:
```js
   <Tabs
      ignoresTopSafeArea={true}
      screenOptions={{
        // ignoresTopSafeArea: true,
      }}>
```

**Comment by ferrannp** [comment]
@okwasniewski I tried `ignoresTopSafeArea: true` and I get the same issue, still seeing wrong bg color.

**Comment by fredrikburmester** [comment]
This prop solved this issue for me.

**Comment by okwasniewski** [comment]
> @okwasniewski I tried `ignoresTopSafeArea: true` and I get the same issue, still seeing wrong bg color.

@ferrannp Yes in the example folder, you can find multiple working examples. One of them ignores safe area insets without issues: https://github.com/okwasniewski/react-native-bottom-tabs/blob/main/example/src/Examples/FourTabs.tsx

If you still encounter issues can you provide a minimal reproduction example?
